Berlin/BerHsOr9834.xml

```diff
@@ -54,12 +54,17 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
                   </objectDesc>
                   <handDesc>
                      <handNote script="Ethiopic" xml:id="h1">
-                          
+
                      </handNote>
                   </handDesc>
                   <decoDesc>
                      <decoNote type="miniature" xml:id="d1">
-                        <desc>Miniaturen</desc>
+                       <locus target="#1v"></locus>
+                        <desc>Miniature of <persName ref="PRS1962Anthony">ʾƎnṭonǝs</persName>.</desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d2">
+                       <locus target="#2r"></locus>
+                        <desc>Miniature of unknown Saint.</desc>
```

Comment:
this is very good, no error
just for the needs of your project you might want to introduce the mark up of art themes, e.g.
`<desc>Miniature of <ref type="authFile" corresp="AT1035HolyMan">unknown Saint</ref>.</desc>
`
and above also 
`<desc>Miniature of <ref type="authFile" corresp="AT1035HolyMan">saint</ref> <persName ref="PRS1962Anthony">ʾƎnṭonǝs</persName>.</desc>`

to make them searchable from art themes menu

Reply:
ah cool, great, yes I will make the additions. 
Also I was so focussed on the right tags that I mixed up the folios. So have to change this anyways too. 
I can make these changes still in the same branch or do I need a new branch?

Reply:
in the same, as long as we have not merged the pull request, this one is active and can be used for new commits